cmd/eval.go

```diff
@@ -96,39 +167,18 @@ func evalCmd() *cobra.Command {
 					}
 				} else {
 					fmt.Println(report)
-					return nil
 				}
-			}
-
-			out := &policy.Output{}
 
-			i, err := image.NewImageValidator(cmd.Context(), arguments.imageRef, arguments.publicKey, arguments.rekorURL)
-			if err != nil {
-				return err
-			}
-
-			if err := i.ValidateImageSignature(cmd.Context()); err != nil {
-				out.SetImageSignatureCheck(false, err.Error())
-				return err
-			}
-			out.SetImageSignatureCheck(true, "success")
-
-			if err := i.ValidateAttestationSignature(cmd.Context()); err != nil {
-				out.SetAttestationSignatureCheck(false, err.Error())
-				return err
-			}
-			out.SetAttestationSignatureCheck(true, "success")
-
-			p, err := policy.NewPolicyEvaluator(arguments.policyConfiguration)
-			if err != nil {
-				return err
+				if arguments.strict && !success {
+					os.Exit(1)
```

Comment:
Let's return error instead, error handling in [HACBS-633](https://issues.redhat.com/browse/HACBS-633) should take care of exiting with the proper status

Reply:
Yes, added with TODO as reminder.